tests/unit/account.test.ts

```diff
@@ -53,10 +53,14 @@ describe("Account", () => {
       const accountAddress = AccountAddress.from(address);
       const newAccount = Account.fromPrivateKeyAndAddress({ privateKey, address: accountAddress, legacy: true });
       expect(newAccount).toBeInstanceOf(Account);
-      expect(newAccount.publicKey).toBeInstanceOf(Ed25519PublicKey);
-      expect(newAccount.privateKey).toBeInstanceOf(Ed25519PrivateKey);
-      expect((newAccount.privateKey as Ed25519PrivateKey).toString()).toEqual(privateKey.toString());
-      expect((newAccount.publicKey as Ed25519PublicKey).toString()).toEqual(new Ed25519PublicKey(publicKey).toString());
+      if (!Ed25519PublicKey.isPublicKey(newAccount.publicKey)) {
```

Comment:
it feels a different testing format - where we throw instead of expect, not sure why we would prefer to throw?